xml/security_ssh.xml

```diff
@@ -1802,4 +1802,110 @@ cd path                            Change remote directory to 'path'
+    <para><emphasis>fail2ban</emphasis>comes with filters for various services, such as Apache,SSH,Courier etc.
+    You can use <emphasis>fail2ban</emphasis> to minimize the rate of incorrect authentications attempts.</para>
+  </sect2>
+  <sect2 xml:id="sec-security-ssh-fail2ban-use">
+    <title>Using <emphasis>fail2ban</emphasis> to stop a SSH brute force attack</title>
+    <para>You can use install and configure<emphasis>fail2ban</emphasis>to protect
+      the remote server against SSH brute force attacks. <emphasis>fail2ban</emphasis>
+      works on authentication log files and by default it scans files such as <filename>/var/log/auth.log</filename>,
+      <filename>/var/log/apache/access.log</filename>, etc.
+    </para>
+    <procedure>
+      <title>Using <emphasis>fail2ban</emphasis> to stop a SSH brute force attack</title>
+      <step><para>To install <emphasis>fail2ban</emphasis>, execute: </para>
+        <screen>&prompt.root;sudo zypper -n in fail2ban firewalld </screen>
+      </step>
+      <step><para>When you install <emphasis>fail2ban</emphasis>, a default configuration file
+        <filename>jail.conf</filename> is also installed. This file gets overwritten
+        when you upgrade <emphasis>fail2ban</emphasis>. To retain any customizations
+        you make to the file, you can copy the <filename>jail.conf</filename> file to
+        another file called <emphasis>jail.local</emphasis>. Both files are automatically read by
+        <emphasis>fail2ban</emphasis>.
+      </para>
+      <screen>&prompt.root;cp /etc/fail2ban/jail.conf /etc/fail2ban/jail.local</screen>
+    </step>
+    <step><para> Open the file in your editor.
+    </para>
+    <screen>vi /etc/fail2ban/jail.local</screen>
+  </step>
+<step><para> Take a look at the four settings that you need to know about.
+</para>
+<figure xml:id="fig-fail2ban-jail-local">
+  <title><emphasis>jail.local </emphasis>file settings</title>
+  <mediaobject>
+   <imageobject role="fo">
+    <imagedata width="75%" fileref="fail2ban.png"/>
+   </imageobject>
+   <imageobject role="html">
+    <imagedata fileref="fail2ban.png"/>
+   </imageobject>
+  </mediaobject>
+ </figure>
+ <variablelist>
+  <varlistentry><term><emphasis>ignoreip</emphasis></term>
+<listitem><para>A list of whitelisted IPs that will never be banned.
+  The localhost IP address <emphasis>127.0.0.1</emphasis> is in the list by default,
+  along with the IPv6 equivalent <emphasis>::1</emphasis>. You can use this setting
+to add a list of IP addresses that you know should not be banned. </para>
+</listitem>
+</varlistentry>
+<varlistentry><term><emphasis>bantime</emphasis></term>
+  <listitem><para>The duration in minutes for which an IP address is banned. When you do not
+    enter a value of <emphasis>m</emphasis> or <emphasis>h</emphasis>, it is treated as seconds.
+    If you enter a value of <emphasis>-1</emphasis>, you are permanently banning an IP address.</para>
```

Comment:
reword this. This sounds like the user does the banning.

Reply:
re-worded